### PR TITLE
Feat/event details

### PR DESCRIPTION
**Pull request by kerry-okpere**

### CODE REVIEW

**Review by AnthoniaOkafor — COMMENTED** [review]
Nice job @kerry-okpere . Location Component already has EventPage added to Router. Wouldn't there be merge conflicts?

**Comment by kerry-okpere** [comment]
Thanks for the heads up I’ll pull from Navigation and see if there are any
conflicts

File: src/components/EventDetail/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./EventDetail.css"
+import Header from "../Header/index";
+import Text from "../Text/index";
+
+const EventDetail = ({about, time, date}) => {
+    return (
+        <aside className="eventDetails">
```
Comment (meekg33k) [inline src/components/EventDetail/index.js:9]: @kerry-okpere, using an `aside` here, this is good use of semantic HTML, really good!

**Review by meekg33k — APPROVED** [review]
Really good stuff @kerry-okpere!

**Pull request merged**